aten/src/ATen/native/quantized/cpu/qnnpack/src/q8conv/4x8-aarch32-neon.S

```diff
@@ -56,10 +95,12 @@ BEGIN_FUNCTION pytorch_q8conv_ukernel_4x8__aarch32_neon
     VMOV.I32 q14, q8
     # Load b_zero_point:
     # - d15 = b_zero_point
-    VLD1.8 {d15[]}, [r9], r4
+    VLD1.8 {d15}, [r5]
+
     # Load a_zero_point:
     # - d14 = a_zero_point
-    VLD1.8 {d14[]}, [r9], r4
+    VLD1.8 {d14[]}, [r9]
+    ADD r9, r9, 4
```

Comment:
If you have a scratch register to spare you can fold the post increment into VLD1.

Reply:
I think there was only one instance of it here. To use r4 I would have to `move 4, #4`. So it was a wash. Earlier it made a difference because there was one more place that did VLD1 so you could save ADD there. Although if you end up having a dependency on r9 then I am not sure what happens. VLD1 has 3/4 cycle latency and if we are doing post-indexing then do you see 3/4 cycles latency on r9 as well? Not clear to me and it depends on when it is incremented and whether there is a bypass path for it or not. I dont think it adds or subtract much from performance anyway.